tests/upload_test.go

```diff
@@ -279,6 +302,40 @@ func uploadFormAsync(portForwardURL, token string, expectedStatus int) error {
 	return uploadToPath(formRequestFunc, portForwardURL, syncFormPath, token, expectedStatus)
 }
 
+func uploadFileNameToPath(requestFunc uploadFileNameRequestCreator, fileName, portForwardURL, path, token string, expectedStatus int) error {
```

Comment:
This is a lot of copy/paste.  Can you get rid of `uploadToPath` or have it call this with the filename?

Reply:
Done, changed the requestFunc to take the extra file name parameter, and now I can use that everywhere.